README.md

```diff
@@ -1,19 +1,289 @@
 # Cypress CircleCI Orb
 
-The Cypress CircleCI Orb is a piece of configuration set in your `circle.yml` file to correctly install, cache and run [Cypress.io](https://cypress.io) tests on CircleCI with very little effort. See this orb in [CircleCI Registry](https://circleci.com/orbs/registry/orb/cypress-io/cypress)
+The [Cypress CircleCI Orb](https://github.com/cypress-io/circleci-orb) is a
+piece of configuration set in your `.circleci/config.yml` file to correctly
+install, cache and run Cypress with very little effort.
 
-## Example
-```yml
-# to use orbs, must use version >= 2.1
+Full documentation can be found at the
+[`cypress-io/circleci-orb`](https://github.com/cypress-io/circleci-orb) repo.
+For the Orb Quick Start Guide and usage cases, view the CircleCI
+[Cypress orb documentation](https://circleci.com/developer/orbs/orb/cypress-io/cypress).
+
+---
+
+## Jobs
+
+### run
+
+A single, complete job to run Cypress tests in your application
+
+## Basic Setup
+
+A typical project can have:
+
+```yaml
 version: 2.1
 orbs:
-    # import Cypress orb by specifying an exact version x.y.z
-    # or the latest version 3.x.x using "@3" syntax
-    cypress: cypress-io/cypress@3
+  # "cypress-io/cypress@3" installs the latest published
+  # version "s.x.y" of the orb. We recommend you then use
+  # the strict explicit version "cypress-io/cypress@3.x.y"
+  # to lock the version and prevent unexpected CI changes
+  cypress: cypress-io/cypress@3
 workflows:
-    build:
-        jobs:
-            # "cypress" is the name of the imported orb
-            # run is the name of the job defined in Cypress orb
-            - cypress/run
-```
\ No newline at end of file
+  build:
+    jobs:
+      - cypress/run # "run" job comes from "cypress" orb
+```
+
+By using the `run` job definitions that Cypress provides inside the orb, it
+brings simplicity and static checks of parameters to your CircleCI
+configuration.
+
+You can find multiple examples at
```

Comment:
This is probably going away, right? Or will there still be examples on the Circle registry?

Reply:
No the examples from the `src/examples` folder will be displayed on this page after we publish

Reply:
See this deploy preview https://circleci.com/developer/orbs/orb/cypress-io/cypress?version=dev:4136de7f3dbd28ba722c91118588231c27d567a6#usage-examples